google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -526,13 +543,23 @@ public Boolean getIsLive() {
         return matchesStorageClass;
       }
 
+      public Integer getDaysSinceNoncurrentTime() {
```

Comment:
Please add documentation to the new methods.

Reply:
done